### ISSUE TITLE
OS.get_datetime_from_unix_time does not work with negative times for dates before epoch

### ISSUE THREAD

**Issue by olivergs** [godotengine/godot#20390]
**Godot version:**
3.0.2

**OS/device including version:**
Linux, Fedora 28 (Did not test on other OSes)

**Issue description:**
OS.get_datetime_from_unix_time does not work with negative times for dates before epoch

**Steps to reproduce:**
Execute the following GDScript code

```
const INITIAL_DATE = -1413072000 # March 23 1925
var datetime = OS.get_datetime_from_unix_time(INITIAL_DATE)
print("The datetime: %s" % datetime)
```
It only prints "The datetime: "

**Comment by mattiascibien** [comment]
https://github.com/godotengine/godot/blob/41b962fc9c5c4517a1665aa4b0f18d486ac49c6a/core/bind/core_bind.cpp#L718-L721 It seems that it should not be possibile to use a negative number as the function accepts only a uint. @akien-mga can you confirm this is the case?

**Comment by akien-mga** [comment]
@mattiascibien That's how it was implemented in 866e47ec by @Razzlegames, but that could be changed.
